colorcoder/Main.java

```diff
@@ -65,40 +65,40 @@ public ColorPair(MajorColor major, MinorColor minor)
             majorColor = major;
             minorColor = minor;
         }
-        public MajorColor getMajor() {
+        public MajorColor getMajorColor() {
             return majorColor;
         }
-        public MinorColor getMinor() {
+        public MinorColor getMinorColor() {
             return minorColor;
         }
-        String ToString() {
-            String colorPairStr = MajorColorNames[majorColor.getIndex()];
+        String getColorPairName() {
+            String colorPairStr = MajorColorNames[majorColor.getColorIndex()];
             colorPairStr += " ";
-            colorPairStr += MinorColorNames[minorColor.getIndex()];
+            colorPairStr += MinorColorNames[minorColor.getColorIndex()];
             return colorPairStr;
         }
     }
 
     static ColorPair getColorFromPairNumber(int pairNumber) {
         int zeroBasedPairNumber = pairNumber - 1;
         MajorColor majorColor = 
-            MajorColor.fromIndex(zeroBasedPairNumber / numberOfMinorColors);
+            MajorColor.fromColorIndex(zeroBasedPairNumber / numberOfMinorColors);
         MinorColor minorColor =
-            MinorColor.fromIndex(zeroBasedPairNumber % numberOfMinorColors);
+            MinorColor.fromColorIndex(zeroBasedPairNumber % numberOfMinorColors);
         return new ColorPair(majorColor, minorColor);
     }
     static int getPairNumberFromColor(MajorColor major, MinorColor minor) {
-        return major.getIndex() * numberOfMinorColors + minor.getIndex() + 1;
+        return major.getColorIndex() * numberOfMinorColors + minor.getColorIndex() + 1;
```

Comment:
Good choice of names :)
we can read this line of code without having to refer back to what was the type of major and minor